server/api/login_handler.py

```diff
@@ -1,7 +1,64 @@
-from flask import render_template, Blueprint
+from flask import render_template, Blueprint, session, jsonify, request
+import json
+import jwt
+import datetime
+
 login_handler = Blueprint('login_handler', __name__)
 
 
-@login_handler.route('/login')
+# Loading sample_users.json as a very simple "database"
+with open('sample_users.json') as sample_users:
+    users = json.load(sample_users)
```

Comment:
wait why is this needed? can you query your Users instead?

Reply:
definitely! was just part of some really old code, before backend was set up